recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+# {% set version = "1.0.2.dev0" %}
+
+package:
+  name: obspy
+  version: {{ version }}
+
+source:
+#  fn: obspy-{{ version }}.zip
+#  url: https://pypi.python.org/packages/source/o/obspy/obspy-{{ version }}.zip
+  fn: condaforge_win.zip
+  url: https://github.com/megies/obspy/archive/condaforge_win.zip
+
+build:
+  number: 0
+  preserve_egg_dir: yes
+  detect_binary_files_with_prefix: true
+  script: python setup.py install
```

Comment:
Since you are using `setuptools` you can change that line to:

`script: python setup.py install --single-version-externally-managed --record record.txt`